src/twisted/web/http.py

```diff
@@ -1190,6 +1195,12 @@ def _ensureBytes(val):
             cookie = cookie + b"; Secure"
         if httpOnly:
             cookie = cookie + b"; HttpOnly"
+        if samesite:
+            samesite = _ensureBytes(samesite).lower()
+            if samesite not in [b"lax", b"strict"]:
+                raise ValueError(
+                    "Invalid value for samesite: " + repr(samesite))
+            cookie += b"; SameSite=" + samesite
```

Comment:
[SameSite-aware clients are case-insensitive to`SameSite`, `Lax`, and `Strict`](https://tools.ietf.org/html/draft-west-first-party-cookies-07#section-4.1).